src/controllers/connectionConfig.ts

```diff
@@ -0,0 +1,67 @@
+import { rejects } from "assert";
+import { resolve } from "path";
+
+const Connection = require('tedious').Connection;
+const RequestStatement = require('tedious').Request;
+let connection;
+
+export function setupConnection(portVal){
+    const config = {
+        "server": "localhost",
+        "authentication": {
+          "type": "default",
+          "options": {
+            "userName": "DanLeb",
+            "password": "TestPassword123"
+          }
+        },
+        "options": {
+          "port": 1433,
+          "database": "bookish",
+          "trustServerCertificate": true
+        }
+      }
+
+    connection = new Connection(config);
+
+    // Setup event handler when the connection is established
+    connection.on('connect', function(err){
+        if(err){
+            console.log("Error: " + err);
+        }
+
+        console.log("Connected successfully");
+    });
+
+    connection.connect();
+}
+
+
+
+export function executeStatement(sqlQuery: string){
+    let request = new RequestStatement(sqlQuery, function(err, rowCount){
+        if(err){
+            console.log(err);
+        } else{
+            console.log(rowCount + " rows");
+        }
+    });
+
+    return new Promise((resolve, reject) => {

```

Comment:
With tedious, you can do something nice by changing the config for this to make it a bit cleaner:
![image](https://user-images.githubusercontent.com/39823505/178265921-2cbd055b-b26c-431b-9b28-a70ee5535a18.png)

Note that you need to set `rowCollectionOnRequestCompletion: true` in your config.